assets/script.js

```diff
@@ -15,23 +15,23 @@ var questions = [
     {
         question: "How does a FOR loop start?",
         choices: ["A. for (i = 0; i <= 5; i++)", "B. for i = 1 to 5", "C. for (i <= 5; i++)", "D. for (i = 0; i <= 5)"],
-        answer: "A. for (i = 0; i <= 5; i++)"
+        answer: 0,
```

Comment:
Doing this to avoid comparing strings. Instead I'm using the index of the correct choice in the array for the comparison. This is
1. faster because the code doesn't have to make sure each character matches
2. safer, if there is an accidental space or special character, they may not truly be equal as expected. More on special character issues here: https://dmitripavlutin.com/compare-javascript-strings/
3. Less repetitive: if you change the text of one of the choices, you don't have to also change the answer field